gpsd_format/drivers.py

```diff
@@ -214,24 +201,39 @@ def read(self, size):
         return self.obj.read(size)
 
 
-class NewlineJSON(BaseDriver):
+class FileDriver(BaseDriver):
```

Comment:
Because some drivers need to let an external library do the file opening, like the lzma and bzip2 modules... There's no way to work around this really :(